module/platform/OpenCR/HardwareIO/src/hardwareio/process_data.cpp

```diff
@@ -50,6 +52,11 @@ namespace module::platform::OpenCR {
         // Button Left = 0x04
         opencr_state.buttons = {bool(data.button & 0x04), bool(data.button & 0x02), bool(data.button & 0x01)};
 
+        // If the left (black) button is pressed trigger field localisation reset
```

Comment:
Make this comment general, its true field localisation is reset in some cases but only if `FieldLocalisation` module is in the role

Reply:
I feel like this comment should not have been in there in the first place hahaha

Reply:
sorted :+1: